vendor/index.yaml

```diff
@@ -430,11 +430,11 @@ vendors:
     ouis:
       - 0002CC
 
-  - id: mcf88
```

Comment:
Hi Franco, do not delete mcf88 this can affect users with this devices, add as a new vendor engiko